app/src/types/api.js

```diff
@@ -65,10 +72,14 @@ export function validateProblemJson(value: any, res: express$Response) {
       message: "Unrecoverable error."
     });
   } else {
-    res.status(value.status).json({
-      // Forward the error received from the API.
-      message: value.detail
-    });
+    if (callback !== null && callback !== undefined) {
+      callback();
+    } else {
+      res.status(value.status).json({
+        // Forward the error received from the API.
+        message: value.detail
```

Comment:
I'm not sure this is a good idea, information may be leaked to the client, I prefer a whitelist approach, only data we know we want to forward to the client gets forwarded